src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Options/FeatureFlagStorageLocation.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.Options
+{
+    /// <summary>
+    /// Specifies that the option is stored in feature flag storage.
+    /// </summary>
+    internal sealed class FeatureFlagStorageLocation : OptionStorageLocation2
+    {
+        public string Name { get; }
+
+        public FeatureFlagStorageLocation(string name, string prefix = "Roslyn.")
```

Comment:
tbh, i find that default prefix more confusing.  i think i'd genuinely just prefer the callsites just write in the full location name there.  if the prefix were actually exposed through this type, or read elsewhere i'd feel differently.  but this is just a way for a small handful of htem to avoid writing `Roslyn.` at the definitino point, and i honestly just found that more confusing.

Reply:
OK.